data/brands/shop/electronics.json

```diff
@@ -1158,6 +1158,21 @@
         "shop": "electronics"
       }
     },
+    {
+      "displayName": "Vanden Borre",
+      "locationSet": {"include": ["be"]},
+      "matchNames": [
+        "Vandenborre",
```

Comment:
Those 3 matchNames only have space and uppercase difference with `brand/name`, so they will be removed when formatter run.

Don't worry, you don't need to do anymore this time.

You can see this intruduction about matchNames: https://github.com/osmlab/name-suggestion-index/wiki/Contributing#matchnamesmatchtags-optional